src/CMakeLists.txt

```diff
@@ -1,14 +1,22 @@
 cmake_minimum_required(VERSION 3.10)
 cmake_policy(SET CMP0077 NEW)
 set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+
 project(VectorSimilarity)
 
 set(CMAKE_CXX_STANDARD 14)
 
+if(NOT DEFINED root)
+	get_filename_component(root ${CMAKE_CURRENT_SOURCE_DIR}/.. ABSOLUTE)
+	set(binroot ${root}/build)
+endif()
+message("VectorSimilarity: " ${binroot})
+
 option(VECSIM_STATIC "Build as static library" OFF)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../deps)
+
+include_directories(${root}/deps)
```

Comment:
I think we can remove it, no? The library is now self contained

Reply:
you do have pybind11 there.
anyway, it will support anything that's added under `deps/` in the future.